common/config/cli.go

```diff
@@ -0,0 +1,124 @@
+package config
+
+import (
+	"bytes"
+	"os"
+	"strconv"
+
+	"gopkg.in/urfave/cli.v1"
+)
+
+const (
+	environmentDev  = "dev"
+	environmentProd = "prod"
```

Comment:
Oh snap I did not realize you refactored the gophrcli tooling to be a go cli program, vry nice.